packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
The entire implementation of ActionList seems too messy at the moment. We're handling RN as well as web specific things here. I think its better if we create `ActionList.native.tsx` & `ActionList.web.tsx`. Both of them have very different implementations anyway, better to keep them isolated. We can share certain things like styling via common functions. What do you think?

Reply:
Yes right, lets refactor this.